<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWdescentWin</class>
 <widget class="QMainWindow" name="CWdescentWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>525</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1" columnstretch="1,1">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Информация о выполнении</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="spacing">
          <number>10</number>
         </property>
         <item row="0" column="1">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Текущая точка</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Новая точка</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Х1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="x1_lb">
           <property name="text">
            <string>я</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="new_x1_lb">
           <property name="text">
            <string>я</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Х2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="x2_lb">
           <property name="text">
            <string>я</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="new_x2_lb">
           <property name="text">
            <string>я</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Значение функции</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="fsign_lb">
           <property name="text">
            <string>я</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="new_fsign_lb">
           <property name="text">
            <string>я</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Расстояние от стартовой до текущей точки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="distance_lb">
           <property name="text">
            <string>ы</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="verticalSpacing">
          <number>10</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Точность</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="precision_lb">
           <property name="text">
            <string>ы</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Шаг по Х1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="x1_step_lb">
           <property name="text">
            <string>ы</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Шаг по Х2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="x2_step_lb">
           <property name="text">
            <string>ы</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Выберите правильное действие</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="calculate_bt">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Вычислить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="6">
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="title">
          <string>Направление движения</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="up_x1_rb">
            <property name="text">
             <string>+ по Х1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="down_x1_rb">
            <property name="text">
             <string>- по Х1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="up_x2_rb">
            <property name="text">
             <string>+ по Х2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="down_x2_rb">
            <property name="text">
             <string>- по Х2</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="accept_bt">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Принять</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="not_accept_bt">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Не принимать</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="end_bt">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Закончить</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="change_step_bt">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Изменить шаг</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListView" name="LogLstVw">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionRectVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
